src/utils/components/DiskModal/utils/helpers.ts

```diff
@@ -320,8 +320,17 @@ export const ejectISOFromCDROM = (vm: V1VirtualMachine, cdromName: string): V1Vi
     );
   });
 };
-export const isHotPluggableEnabled = (featureGates: string[]) => {
+
+export const isDeclarativeHotplugVolumesEnabled = (featureGates: string[]) => {
   if (featureGates?.includes(HotPlugFeatures.DeclarativeHotplugVolumes)) {
+      return true;
+  }
+  return false;
+};
+
+
+export const isHotPluggableEnabled = (featureGates: string[]) => {
+  if (isDeclarativeHotplugVolumesEnabled(featureGates)) {
```

Comment:
Looking at [our past discussion](https://github.com/kubevirt-ui/kubevirt-plugin/pull/3005#discussion_r2362881414) it seems we need only to rename `isHotPluggableEnabled -> isDeclarativeHotplugVolumesEnabled`.